src/GafferRenderManModule/GafferRenderManModule.cpp

```diff
@@ -56,28 +54,12 @@ using namespace GafferRenderMan;
 namespace
 {
 
-/// \todo Move this serialisation to the bindings for GafferScene::Shader, once we've made Shader::loadShader() virtual
-/// and implemented it so reloading works in OpenGLShader. Also consider how we might share loading code
-/// between shaders and lights.
-class RenderManShaderSerialiser : public GafferBindings::NodeSerialiser
-{
-
-	virtual std::string postScript( const Gaffer::GraphComponent *graphComponent, const std::string &identifier, const Serialisation &serialisation ) const
-	{
-		const RenderManShader *shader = static_cast<const RenderManShader *>( graphComponent );
-		std::string shaderName = shader->namePlug()->getValue();
-		if( shaderName.size() )
-		{
-			return boost::str( boost::format( "%s.loadShader( \"%s\", keepExistingValues=True )\n" ) % identifier % shaderName );
-		}
-
-		return "";
-	}
-
-};
 
 void loadShader( RenderManShader &shader, const std::string &shaderName, bool keepExistingValues )
 {
+	// Keeping a custom binding of loadShader for RenderManShader because the default one
```

Comment:
Nope!  I was just trying to change as little behaviour as possible during the refactor.  I can add it to the default one and remove this.